tfjs-backend-webgpu/src/kernels/webgpu_program.ts

```diff
@@ -94,3 +104,9 @@ export function makeShaderKey<R extends Rank>(
       inputShapesEqualsOutShape + program.shaderKey;
```

Comment:
I think no need for this. For any program,  if it uses WGSL, it will not use GLSL.

Reply:
But if the first time, the user uses wgsl, and the second time the user uses glsl. I think the shaders won't be recompiled since they see the same shader key.

Reply:
For this scenario, you are right, updated, ptal.